templates/institutional/_base.scss

```diff
@@ -0,0 +1,6 @@
+/* Institutional landing page background */
+.provisional.gc-institutional-bg {
```

Comment:
Is this should not have been converted into component instead?

Template CSS might require to use a special body class